doc/man/8/cephfs-shell.rst

```diff
@@ -164,6 +164,23 @@ Usage :
 
 * file - name of the file
 
+ln
+--
+
+Create a hard link or a symbolic link to an existing file or directory.
```

Comment:
Suggest:
"Create a hard or symbolic link to an existing file or directory"

Do we allow hard links to directories?

Reply:
Ah, nope. Hard links are no allowed for directories. Should I change it to what man page of `ln` says - "make links between files"?

Reply:
Ack.  In 1988 I forced a hard link to a UFS directory and learned my lesson ;)

I suggest being explicit:

"Add a hard link to an existing file or create a symbolic link to an existing file or directory."

Reply:
Done. Thanks!